docker-compose.test.yaml

```diff
@@ -17,11 +17,15 @@ services:
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
     environment:
-      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-submission-azurite:10000/devstoreaccount1;
+      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-dax-azurite:10000/devstoreaccount1;
   
   ffc-pay-submission-postgres:
     volumes:
       - postgres_data:/var/lib/postgresql/data
+  
+  ffc-pay-dax-azurite:
+    labels:
+      com.docker.compose.pay.role: test
```

Comment:
This label means when checking if storage exists, test containers will be ignored.

Labels follow compose convention